app/controllers/admin/users.js

```diff
@@ -0,0 +1,11 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+
+export default Controller.extend({
+  routing        : Ember.inject.service('-routing'),
```

Comment:
Isn’t this injected globally already?

Reply:
If I remove this I get wrong results. And I tried printing the value of currentRouteName, it is undefined.